release-calendar/src/server/api-service.ts

```diff
@@ -27,4 +28,8 @@ export class ApiService {
   async getReleases(): Promise<Release[]> {
     return this.releaseRepository.find();
   }
+
+  async getChannel(desiredChannel: Channel): Promise<Release[]> {
```

Comment:
Does this belong in this PR?

Reply:
oops, nope, that's for later. I'll take it out for now